Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/TicketService.hs

```diff
@@ -394,14 +458,22 @@ getTicketBookingsDetails _ shortId_ = do
           { peopleCategories = peopleCategoryDetails,
             amount = amount.amount,
             amountWithCurrency = mkPriceAPIEntity amount,
+            amountToRefund = amountToRefund,
+            bookedSeats = bookedSeats - fromMaybe 0 cancelledSeats,
```

Comment:
If any seat is cancelled, we are already subtracting it from `bookedSeats`?

Reply:
now we are not subtracting the bookedSeats, as it might change the sacred truth of booked seats

Reply:
but yes, i think we shouldn't subtract it, send this to frontend to info with cancelled seats